FluentTerminal.App.ViewModels/MainViewModel.cs

```diff
@@ -283,25 +281,25 @@ public async Task AddSavedShhRemoteTerminal()
             }
             else
             {
-                await ApplicationView.RunOnDispatcherThread(() => AddTerminal(profile));
+                await AddTerminalAsync(profile);
```

Comment:
are you sure about removing the Dispatcher here? There might be a case were this was necessary

Reply:
Yes, I'm sure. As you can see if you follow the calls, in the last method in the stack `AddTerminalAsync()` uses the dispatcher.
So far we had a situation when we are calling dispatcher several times for the same task, meaning that we are scheduling dispatcher task that schedules dispatcher task to do something...

Reply:
At the end of the day, you can test in my branch.